src/TwitterStream.d.ts

```diff
@@ -1,5 +1,5 @@
 export class TwitterStream<T> implements AsyncIterable<T> {
-  constructor(connect: () => Promise<any>, close: () => void);
+  constructor(connect: () => Promise<any>, close: () => void, options: object);
```

Comment:
Let's declare the `options` type instead of making it an opaque object.

Reply:
will the following suffice?
```
export interface StreamOptions {
    [key: string]: any;
}

export class TwitterStream<T> implements AsyncIterable<T> {
  constructor(connect: () => Promise<any>, close: () => void, options: StreamOptions);

  [Symbol.asyncIterator](): AsyncIterator<T>;

  public close(): void;
}
```

Reply:
```ts
export interface StreamOptions {
  // Number of seconds to wait for data or a heartbeat ping from Twitter before considering the stream closed.
  timeout: number | undefined;
}
```

and while we're at it, can we increase the default timeout to 30 seconds as per #64?